source/common/runtime/runtime_features.cc

```diff
@@ -67,7 +67,6 @@ RUNTIME_GUARD(envoy_reloadable_features_postpone_h3_client_connect_to_next_loop)
 RUNTIME_GUARD(envoy_reloadable_features_quic_defer_send_in_response_to_packet);
 RUNTIME_GUARD(envoy_reloadable_features_skip_delay_close);
 RUNTIME_GUARD(envoy_reloadable_features_strict_check_on_ipv4_compat);
-RUNTIME_GUARD(envoy_reloadable_features_support_locality_update_on_eds_cluster_endpoints);
```

Comment:
Do you want to mention this change under "Removed config or runtime" changelog ?